Configure

```diff
@@ -132,15 +132,19 @@ my $gcc_devteam_warn = "-DDEBUG_UNUSED"
         . " -Werror"
```

Comment:
Just in case, if it wasn't clear, my remark here was specifically about icc.

Reply:
That is how I interpreted it: icc builds add a `-Wno-swtich` open to the command line.  The other compilers don't.

It seems like we've reached an acceptable resolution, thanks @levitte and @dot-asm 

It might be interesting to see some performance comparisons between the three compilers. 
 I'd bet against the C compilers coming remotely close to the asm code but the framework around that might show significant differences.